build-cluster-ec2.yml

```diff
@@ -106,14 +207,37 @@
   tasks:
     - include: python/tasks/main.yml
     - include: java/tasks/main.yml
+    - include: ntp/tasks/main.yml
     - include: docker/tasks/main.yml
       when: "{{ INSTALL_DOCKER }} == true"
+    - include: conductr/tasks/install-agent.yml
+    - include: bundle/tasks/config-elasticsearch.yml
+
+- name: Setup Public Nodes
+  hosts: proxy_nodes
+  user: "{{ REMOTE_USER }}"
+  gather_facts: False
+  sudo: True
+
+  vars:
+# These are the default ConductR Agent roles - these roles may be overridden within the VARS_FILE
+    CONDUCTR_AGENT_ROLES:
```

Comment:
Can we rename `CONDUCTR_AGENT_ROLES` to `CONDUCTR_PUBLIC_AGENT_ROLES`?

Similarly the comment needs to be updated to `These are the default ConductR Public Agent roles - - these roles may be overridden within the VARS_FILE`

Once this is done, we can then modify https://github.com/typesafehub/conductr-ansible/pull/34/files#diff-a1aa4b9835ceeb0000cc881144e3762aR237

This is to allow user the ability to set different roles for private vs public nodes.